### PR TITLE
6pm d show current quarter

### PR DESCRIPTION
**Pull request by ssmith563**
This PR fixes issue #32.

As a user who is viewing the courses table, I can only see information for the current quarter so that I do not get confused by previous or future classes.

**Acceptance Criteria**
 - Courses table should only show info for the current quarter.
 - Info should include: Course ID, Course name, Current quarter, first name, last name, and email

Previously, the courses table shows courses for all quarters. This PR changes this so that a user viewing the courses table can only see information for the current quarter so that they do not get confused by previous or future classes.

### CODE REVIEW

**Comment by andrewhlu** [comment]
Looks like you're just short of 100% test coverage right now. Would you mind addressing this and re-requesting the staff tomorrow?

**Comment by pconrad** [comment]
Please improve the description.  

```
Fixes issue 32
```

is not enough.  First, please use #32 so that it links to the issue.  Second please copy the user story into the issue so that the reader knows at a high level what the PR is addressing without having to go hunting for the issue.

Finally, fix the test coverage.

File: src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java [hunk 1]
```diff
@@ -91,9 +92,18 @@
   @GetMapping(value = "/api/public/courses", produces = "application/json")
   public ResponseEntity<String> getCourses() throws JsonProcessingException {
     List<Course> courseList = courseRepository.findAll();
+
+    String currentQuarter = "F20"; //This should be deleted once configuration file is implemented
+
+    List<Course> currentCourses = new ArrayList<Course>();
+    for(Course i: courseList){
+      if(i.getQuarter().equalsIgnoreCase(currentQuarter)){
```
Comment (CharityHsu) [inline src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java:100]: On Codecov it looks like line 100 is yellow, so might need to write a test that makes the if statement true

**Review by CharityHsu — CHANGES_REQUESTED** [review]
Looks good, but try adding another test because to get full coverage the if condition needs to be performed twice, for a true case and a false case. I think you might need a test where the course quarter is not the current quarter.

**Review by CharityHsu — DISMISSED** [review]
Looks good, nice job figuring it out

**Comment by pconrad** [comment]
Closing without merging---other teams' code have made this PR obsolete.  (The feature is already in the code base in an improved form, i.e. with setting the current quarter in the app rather than having it hard coded.)

**Pull request closed without merging**